common/errors.js

```diff
@@ -146,45 +146,41 @@
     .factory('ErrorService', ['AlertsService', 'errorNames', 'Session', '$log', '$rootScope', '$window', 'errorMessages', 'Errors', 'DataUtils', 'UriUtils', 'modalUtils',
           function ErrorService(AlertsService, errorNames, Session, $log, $rootScope, $window, errorMessages, Errors, DataUtils, UriUtils, modalUtils) {
 
-        var reloadCb = function(){
+        var reloadCb = function() {
             window.location.reload();
         };
 
-        function errorPopup(message, errorStatus, pageName, redirectLink, subMessage, stackTrace, errorCode, isDismissible) {
-            var providedLink = true,
-                isLoggedIn = false;
+        function errorPopup(message, errorStatus, pageName, redirectLink, subMessage, stackTrace, errorCode, isDismissible, showLogin) {
```

Comment:
Based on my other comment on the `modal.js`, if we pass the actual object then the `message` and `errorStatus` seem more like optional parameters to this functions. Because only in one case we're actually changing them. So we can move them to the end of the argument list and they can be undefined.